### PR TITLE
Remove unused variables

### PR DESCRIPTION
**Pull request by fredj**
Found by [ESLint](http://eslint.org/)

### CODE REVIEW

File: src/lib/style-extends.html [hunk 1]
```diff
@@ -25,7 +25,6 @@
       var rules = styleUtil.rulesForStyle(style);
       var self = this;
       styleUtil.forEachStyleRule(rules, function(rule) {
-        var map = self._mapRule(rule);
```
Comment (JeremybellEU) [inline src/lib/style-extends.html]: This changes seems to have caused a failure on Travis https://travis-ci.org/Polymer/polymer/jobs/88971521#L2290. This line should probably be `self._mapRule(rule)` (note the removed assignment to `map`).

Make sure you run the test suite before submitting a Pull-request. #2658 has a pending explanation on how you can do this.

Comment (fredj) [inline src/lib/style-extends.html]: Thanks for the hint

**Comment by fredj** [comment]
Thanks for any review

**Comment by sorvell** [comment]
Sorry for the delay and thanks for your effort. Please rebase this PR against master and we'll merge it if there are still issues.

**Comment by fredj** [comment]
@sorvell rebased but the travis build failed (I don't think it's related to my changes)

**Comment by TimvdLippe** [comment]
@fredj The build failed due to another PR as described in issue #3371

**Comment by fredj** [comment]
@TimvdLippe ok, thanks

**Comment by fredj** [comment]
@sorvell rebased

**Comment by azakus** [comment]
These changes were also made in #3382.

**Pull request closed without merging**